server/model/User.php

```diff
@@ -5,27 +5,27 @@ class User
     private $id;
     private $firstName;
     private $lastName;
-    private $userRole;
+    private $role;
     private $phone;
     private $email;
     private $registered;
     private $lastLogin;
     private $wantsSMS;
-    private $wantsEmails;
+    private $wantsEmail;
 
     public function getUserInfo()
     {
         return [
             "id" => $this->id,
             "firstName" => $this->firstName,
             "lastName" => $this->lastName,
-            "userRole" => $this->userRole,
+            "userRole" => $this->role,
```

Comment:
I like that you left the API alone. I'll be able to update the frontend in a different PR.